client/components/forms/form-select/index.jsx

```diff
@@ -1,19 +1,28 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	classnames = require( 'classnames' ),
-	omit = require( 'lodash/omit' );
+import React from 'react';
+import classNames from 'classnames';
+import omit from 'lodash/omit';
 
-module.exports = React.createClass( {
+const FormSelect = React.createClass( {
+	getDefaultProps() {
+		return {
+			isError: false,
+		};
+	},
```

Comment:
Technically, this isn't necessary since undefined is a falsy value. You don't need to remove this but I was wondering why you wanted to add it